### ISSUE TITLE
Support for workspace edition

### ISSUE THREAD

**Issue by ppershing** [kbknapp/cargo-outdated#336]
`cargo outdated` currently fails to parse workspace members containing
`edition.workspace = true` (which is equivalent to `edition = {workspace = true}`
And fails with quite obvious error
```
error: failed to load manifest for workspace member `<redacted>`

Caused by:
  failed to parse manifest at `<redacted>`

Caused by:
  invalid type: map, expected a string for key `package.edition`
```

How hard would it be to add support for using workspace edition?

**Comment by ewoolsey** [comment]
Chiming in! Also experiencing the same thing, would love to see an update!

**Comment by ogoffart** [comment]
I think this will get solved by updating the cargo dependency.
There are several PR that does that.  Eg https://github.com/kbknapp/cargo-outdated/pull/329

I wonder if this repository is still maintained.

**Comment by dns2utf8** [comment]
Yes it is all the `*.workspace = true` keys. Would love to see it

**Comment by GeeWee** [comment]
It does not seem solved still on the latest version.

**Comment by davidhewitt** [comment]
I believe this will have been solved by cargo updates since the issue was reported.
